docs/cli.md

```diff
@@ -368,6 +366,127 @@ Output will look like this, where `python` will be green as it is the package th
  xz                5.2.6       h166bdaf_0          408.6 KiB  conda  xz-5.2.6-h166bdaf_0.tar.bz2
 ```
 
+## `tree`
+
+Display the project's packages in a tree. Highlighted packages are those specified in the manifest.
+
+The package tree can also be inverted (`-i`), to see which packages require a specific dependencies.
+
+##### Arguments
+
+- `REGEX` optional regex of which direct dependencies to filter the tree to, or which dependencies to start with when inverting the tree.
+
+##### Options
+
+- `--invert (-i)`: Invert the dependency tree, that is given a `REGEX` pattern that matches some packages, show all the packages that depend on those.
+- `--platform <PLATFORM> (-p)`: The platform to list packages for. Defaults to the current platform
+- `--manifest-path <MANIFEST_PATH>`: The path to [manifest file](configuration.md), by default it searches for one in the parent directories.
+- `--environment`(`-e`): The environment's packages to list, if non is provided the default environment's packages will be listed.
```

Comment:
```suggestion
- `--environment (-e)`: The environment's packages to list, if non is provided the default environment's packages will be listed.
```

we should also make this consistent in `list` (and all other occurrences that i didn't spot)